app/views/settings/settings.html

```diff
@@ -16,6 +16,18 @@
                 </div>
             </div>
         </li>
+        <li class="full-flex" ng-hide="!showTraySettings">
```

Comment:
if `ng-if` is used, variable in `ng-model` or `ng-checked` won't change for some reason.